assets/pcw-hugo-theme/css/custom.css

```diff
@@ -1,82 +1,51 @@
-/*********
-* COLORS *
-*********/
+/************************
+* Specific Item Targets *
+*************************/
 
-:root {
-  --orange: rgb(255, 128, 0);
-
-  --blue: #5262ed;
-  --lilac: #d48aff;
-  --dark-blue: #123248;
-  --light-gray: #e6e6e6;
-
-  --primary: var(--blue);
+/* Hero Image */
+  /* 
+  Extracted from inline HTML style 
+  */
+div.hero-image a {
+  border: dashed 1px white;
 }
 
-/* Background color */
-.bg-pcw {
-  background-color: var(--primary) !important;
-  color: white !important;
-}
-.bg-pcw-lilac {
-  background-color: var(--lilac) !important;
-  color: white !important;
-}
-.bg-pcw-dark-blue {
-  background-color: var(--dark-blue) !important;
-  color: var(--light-gray) !important;
-}
-.bg-light-gray {
-  background-color: var(--light-gray) !important;
-  color: black !important;
+/* Site Navigation */
+  /*
+  Giving the nav items some breathing room
+  */
+nav a { 
+  padding: 0.2rem;
 }
```

Comment:
Added this padding bc I wanted to add a border to the nav text when focused and the borders look terrible w/out padding. Will affect layout a tiny bit but I think the effect is pretty tiny.